test/gtest/ucs/test_vfs.cc

```diff
@@ -106,3 +107,19 @@ UCS_TEST_F(test_vfs_sock, send_recv_nop) {
     do_send_recv(UCS_VFS_SOCK_ACTION_NOP, m_sockets[0], m_sockets[1], -1,
                  &msg_out);
 }
+
+class test_vfs_obj : public ucs::test {
+public:
+    static void show_info(void *obj, ucs_string_buffer_t *strb)
+    {
+        ucs_string_buffer_appendf(strb, "info");
+    }
+};
+
+UCS_TEST_F(test_vfs_obj, add_remove_obj) {
+    char obj;
+
+    ucs_vfs_obj_add_dir(NULL, &obj, "obj");
```

Comment:
can you please add a couple of more tests, e.g.:
adding simple tree of objects, removing some objects in the middle, etc

Reply:
updated gtests